site/docs/extensions/index.md

```diff
@@ -0,0 +1,38 @@
+# Extensions
+
+In many cases, the existing objects in Substrait will be sufficient to accomplish a particular use case. However, it is sometimes helpful to create a new data type, scalar function signature or some other custom representation within a system. For that, Substrait provides a number of extension points. 
+
+## Simple Extensions
+
+Some kinds of primitives are so frequently extended that Substrait defines a standard YAML format that describes how the extended functionality can be interpreted. This allows different projects/systems to use the YAML definition as a specification to interoperate around the extended items so that interoperability isn't constrained solely the base Substrait specification. The main types of extensions that are defined in this manner include the following:
```

Comment:
```suggestion
Some kinds of primitives are so frequently extended that Substrait defines a standard YAML format that describes how the extended functionality can be interpreted. This allows different projects/systems to use the YAML definition as a specification so that interoperability isn't constrained to the base Substrait specification. The main types of extensions that are defined in this manner include the following:
```